src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/ET1ServingService.java

```diff
@@ -0,0 +1,78 @@
+package uk.gov.hmcts.ethos.replacement.docmosis.service;
+
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.et.common.model.ccd.CaseData;
+import uk.gov.hmcts.et.common.model.ccd.items.DocumentTypeItem;
+import uk.gov.hmcts.et.common.model.ccd.items.RespondentSumTypeItem;
+import uk.gov.hmcts.et.common.model.ccd.types.ClaimantIndType;
+
+import java.text.MessageFormat;
+import java.util.List;
+
+@Service
+public class ET1ServingService {
+    public static final String SERVING_DOCUMENT_OTHER_TYPE = "Another type of document";
+    private static final String SERVING_RECIPIENT_CLAIMANT = "Claimant";
+    private static final String SERVING_RECIPIENT_RESPONDENT = "Respondent";
+    private static final String ACAS_MAILTO_LINK = "mailto:ET3@acas.org.uk?subject={0}&body=Parties%20in%20claim" +
+            "%3A%20{1}%20vs%20{2}%0D%0ACase%20reference%20number%3A%20{3}%0D%0A%0D%0ADear%20Acas%2C%0D%0A%0D%0AThe%" +
+            "20tribunal%20has%20completed%20ET1%20serving%20to%20the%20respondent.%0D%0A%0D%0AThe%20documents%20we" +
+            "%20sent%20are%20attached%20to%20this%20email.%0D%0A%0D%0A";
+
+    public String generateOtherTypeDocumentName(List<DocumentTypeItem> docList) {
+        StringBuilder sb = new StringBuilder();
+        for (DocumentTypeItem doc : docList) {
+            if (doc.getValue().getTypeOfDocument().equals(SERVING_DOCUMENT_OTHER_TYPE)) {
+                sb.append("**<big>");
+                sb.append(doc.getValue().getUploadedDocument().getDocumentFilename());
+                sb.append("</big>**<br/>");
+                sb.append("<small>");
+                sb.append(doc.getValue().getShortDescription());
+                sb.append("</small><br/>");
+            }

```

Comment:
I question whether this would be better handled as a string template.  I'm happy either way, but just thought I'd raise the question all the same ;)

Reply:
Refactored to use string format for the readability.